INSTALL.md

```diff
@@ -12,6 +12,12 @@ of the corresponding instructions in Section 1.
 
 *We recommend that you not bother with Section 5 the first time you build RandLAPACK.*
 
+## 0. Software requirements
+RandLAPACK_GPU temporary requirements:
+GNU 13.1.0
+NVIDIA 12.4.131 (make sure to use driver v 550)
+CMAKE 3.29.2
+All that is used to ensure we can compile with C++20 features with no issues.
```

Comment:
Change CMake spec to 3.27.

Also, make sure RandLAPACK can compile even if CUDA isn't present. Obviously you can't do anything GPU-based, but the CPU-only functionality should still work.

Reply:
Nah, that did not fix it.
I can open a git issue on that.

Reply:
Not working without the GPU support rn, I'll need to figure out a way for that,